docs/book/stacks-and-components/component-guide/experiment-trackers/wandb.md

```diff
@@ -185,7 +185,7 @@ from zenml.client import Client
 
 last_run = client.get_pipeline("<PIPELINE_NAME>").last_run
 trainer_step = last_run.get_step("<STEP_NAME>")
-tracking_url = trainer_step.run_metadata.get("experiment_tracker_url")
+tracking_url = trainer_step.run_metadata["experiment_tracker_url"].value
 print(tracking_url.value)
```

Comment:
This code was valid - we do `print(tracking_url.value)` below and `.get` is a valid dict operation.